src/api/customerInstitutionsApi.ts

```diff
@@ -32,29 +33,12 @@ export const getAllCustomerInstitutions = async () => {
 };
 
 export const getCustomerInstitution = async (identifier: string, cancelToken?: CancelToken) => {
-  try {
-    const idToken = await getIdToken();
-    const headers = {
-      Authorization: `Bearer ${idToken}`,
-    };
-    const response = await Axios.get(`${CustomerInstitutionApiPaths.CUSTOMER_INSTITUTION}/${identifier}`, {
-      headers,
-      cancelToken,
-    });
-    if (response.status === StatusCode.OK) {
-      return response.data;
-    } else {
-      return {
-        error: i18n.t('feedback:error.get_customer'),
-      };
-    }
-  } catch (error) {
-    if (!Axios.isCancel(error)) {
-      return {
-        error: i18n.t('feedback:error.get_customer'),
-      };
-    }
-  }
+  const response = await authenticatedApiRequest({
+    url: `${CustomerInstitutionApiPaths.CUSTOMER_INSTITUTION}/${identifier}`,
+    method: 'GET',
```

Comment:
kanskje best å lage enums til disse metodene?

Reply:
nødvendig med `const response`?

Reply:
apiRequest tar inn `AxiosRequestConfig` som har typer: https://github.com/axios/axios/blob/4b3947aa59aaa3c0a6187ef20d1b9dddb9bbf066/index.d.ts#L44

Kan godt returnere direkte ja (uten response). Hvis vi flytter feilmeldinger til komponentene spørs det om vi egentlig trenger disse ``xxxApi`-filene da, siden det eneste som ligger der er URL til endpoint. Men tenkte de kunne være greit å håndtere URLer utenfor komponentene, i tillegg kan vi gjøre ekstra jobb på data som sendes til/fra API her, og også koble sammen flere kall, feks hent ressurs for så å oppdater den (som vi gjør når vi legger til roller nå). Hva tror du?

Reply:
...men vi kan vel fortsatt ha enums eller consts på method? Det er jo bare en streng i `Method` typen?

Ja, enten kan vi gjøre det sånn som du sier, ellers kan vi ha en fil med alle urls? 

Men det må nok mest sannsynlig gjøres noe prosessering av data i noen api-filer, så kan kanskje la de være for nå? Så tar vi det som det kommer?

Reply:
Får feil med en ugyldig method da, så personlig syns jeg det en enklere med `"GET"` enn `Method.GET`.

`Method`-typen ser slik ut:
```
export type Method =
  | 'get' | 'GET'
  | 'delete' | 'DELETE'
  | 'head' | 'HEAD'
  | 'options' | 'OPTIONS'
  | 'post' | 'POST'
  | 'put' | 'PUT'
  | 'patch' | 'PATCH'
  | 'link' | 'LINK'
  | 'unlink' | 'UNLINK'
```

![bilde](https://user-images.githubusercontent.com/6313468/90102465-efdb6600-dd40-11ea-9c73-0bb39f90fc4d.png)

Reply:
hvis det kommer noe ekstrabehandlig av data en plass er det greit om det er likt overalt ja, så vi kan prøve det slik for nå

Reply:
jeg foretrekker `Method.GET` 😛  men da får du bestemme da siden du eier PRen her!